### PR TITLE
KBC-2828 add snowflake add column on typed tables

### PR DESCRIPTION
**Pull request by zajca**
Jira: KBC-2828
KBC: https://github.com/keboola/connection/pull/3914

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -3290,7 +3290,7 @@ attributes and information about the containing bucket.
 ### Add Column to Table [POST]
 Adds a new column to an existing table. This request is [asynchronous](#introduction/synchronous-and-asynchronous-calls).
 
-*Attribute definition is allowed (and required) for typed tables created via [table definition](#reference/tables/create-table-definition)* with Synapse or Exasol backend only
+*Attribute definition and basetype is allowed (and required) for typed tables created via [table definition](#reference/tables/create-table-definition)* with Synapse, Snowflake and Exasol backend only
```
Comment (martinjunger) [inline apiary.apib:3293]: Tady to chce upresnit, pac ted je podpora basetype u pridani sloupce jen pro Snowflake.
Synapse a Exasol to jeste nemaji.

**Review by martinjunger — CHANGES_REQUESTED** [review]
OK, jen upresnit dokumentaci.

File: apiary.apib (hunk 1 above)
Comment (zajca) [reply apiary.apib:3293]: Majů to aj Exasol aj Synapse

**Review by martinjunger — APPROVED** [review]
OK

**Pull request merged**